packages/app-builder/src/tailwind.css

```
@@ -54,3 +54,5 @@
 @utility animate-ping-once {
   animation: ping-once 1s cubic-bezier(0, 0, 0.2, 1);
 }
+
+@custom-variant sentinel-intersect ([data-sentinel][data-intersect] + &);
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)